tip-0007.md

```diff
@@ -0,0 +1,35 @@
+    TIP:  0007 
+    Title:  The First Sustainable Blockchain
+    Authors: Sukesh Kumar Tedla, sukeshtedla@telosgreen.com
+    Status: Draft
+    Type: Informational
+    Created: 2018-09-01
+    Discussion: TIP channel in slack
+    
+# Abstract
+This proposal outlines a plan to make Telos-The First Sustainable Blockchain in the market. Based on research it's estimated that bitcoin network alone consumes 1% of total power consumption in the world which is quite huge and it's just waste of energy, bad to the environment. It's important to think about the impacts of our actions on society, nature and future generation. During this year the world has seen the rise in temperatures like they have never seen before. Though a long summer seems to be nice but we have to realise that increase in temperatures have a drastic effort on the environment.
+
+# Motivation
+We all agree that blockchain is a great innovative technology of the current generation and it's here to stay. But we have to at some point in time realize, what price we are paying for this technology. The POW blockchains have proven to be de-centralised, secure but at the same time they are not efficient [1]. When it comes POS and DPOS blockchains we still have a certain level of decentralization and better efficiency but as they scale they can become un-sustainable as well. EOS have proven to be better when compared to other blockchains but it still have its own issues to address.
+As Telos is driving towards a more decentralised blockchain and addressing the concerns of EOS, it has already started the work in the direction of becoming the top-notch blockchain in the market. This proposal will hopefully take it to the next level and make Telos-The first sustainable blockchain in the world.
+
+# Rational
+We have BP requirements in Telos and we are on a path to do something great. So this proposal puts some additional requirements on BPs and Telos Network to address the Sustainable Development Goals put forth by the United Nations. If we adapt this proposal and live by its standards, it can attract the real industries and applications to use Telos. The organisations, applications can get an extra accreditation for working towards SDGs and using Telos infrastructure.
+
+# Specifications
+It’s divided into 2 sections:
+1)	BP – Requirements: All the BPs of Telos network are required to use Green energy before we reach 1 year from the mainnet activation date. If there are no sources for Green energy usage in the regions where BPs are located they can suffice by addressing other similar initiatives.
```

Comment:
What are the sources of Green energy? Maybe we need to define them? Otherwise maybe people do not know what are Green Energy Resources?